topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -2,34 +2,29 @@ package strings
 
 func ReverseString(str string) string {
 
-	// Here we create a slice of codePoints.
-	codePoints := []rune(str)
+	// Here we create a slice of r.
+	r := []rune(str)
 
-	// Here we create int that will be a pointer to the front of the codePoints.
+	// Here we create int that will be a pointer to the front of the r.
 	var beg int
 
-	// Here  we create int that will be a pointer to the end of the codePoints.
-	end := len(codePoints) - 1
+	// Here  we create int that will be a pointer to the end of the r.
+	end := len(r) - 1
 
 	// While there are still code points to check.
 	for beg < end {
 
-		// Swap the code points by:
+		// Swap the code point.
+		c := r[beg]
```

Comment:
then this can be `r := runes[beg]`

Reply:
This is a complicated program for naming things.

Reply:
Completed